src/gui/qgsfontbutton.cpp

```diff
@@ -957,7 +1093,7 @@ void QgsFontButton::updatePreview( const QColor &color, QgsTextFormat *format, Q
 void QgsFontButton::copyColor()
 {
   //copy color
-  QApplication::clipboard()->setMimeData( QgsSymbolLayerUtils::colorToMimeData( mFormat.color() ) );
+  QApplication::clipboard()->setMimeData( QgsSymbolLayerUtils::colorToMimeData( currentColor() ) );
```

Comment:
From the docs currentColor only works in qfontcolor mode, so this would cause a regression for text format mode.